.github/workflows/trivy-scan-template.yml

```diff
@@ -43,10 +43,10 @@ jobs:
         run: mkdir -p ballerina/lib
 
       - name: Run Trivy vulnerability scanner
-        uses: aquasecurity/trivy-action@0.23.0
+        uses: aquasecurity/trivy-action@master
         with:
           scan-type: "rootfs"
-          scan-ref: "/github/workspace/ballerina/lib"
+          scan-ref: "${{ github.workspace }}/ballerina/lib"
           format: "table"
           timeout: "10m0s"
           exit-code: "1"
```

Comment:
```suggestion
          exit-code: "1"
          scanners: "vuln"
```

Shall we limit to scan only vulnerabilities? otherwise it scan secrets and all?